include/match.h

```diff
@@ -0,0 +1,153 @@
+#ifndef INCLUDE_MATCH_H_
+#define INCLUDE_MATCH_H_
+
+/* Match struct */
+typedef struct {
+    char* word;
+    int line;
+} match;
+
+/* 
+ * Creates new match 
```

Comment:
This header comment style conflicts with Max's in the `ftsh` sources. Please standardize on one of these styles. I don't care which as long as it's consistent.